### ISSUE TITLE
Restart multipass will result in State: Unknown (and machine_type is empty)

### ISSUE THREAD

**Issue by sanderhahn** [canonical/multipass#1383]
**Environment**
```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

$ multipass version
multipass  1.0.2
multipassd 1.0.2
```

**Issue after restart**
Launching a bionic image succeeds:

```bash
multipass launch --name xyz bionic
```
However when restarting `multipassd` the vm will fail to start (State becomes Unknown) and an error is logged:

```bash
sudo snap restart multipass
journalctl -afu snap.multipass*
```

Error in the logs:

```
multipassd[3913]: qemu-system-x86_64: warning: Unknown firmware file in legacy mode: etc/msr_feature_control
                  qemu-system-x86_64: Machine type received is 'pc-i440fx-bionic' and local is 'pc-i440fx-xenial'
                  qemu-system-x86_64: Error -22 while loading VM state
```

**Workaround**
The workaround seems to be to fill the empty `machine_type` for the vm in `/var/snap/multipass/common/data/multipassd/multipassd-vm-instances.json` with the value `pc-i440fx-bionic` and restarting `multipass` again.

Thank you for `multipass` very nice tooling :)

**Comment by Saviq** [comment]
Hi @sanderhahn, this is actually fixed now in #1304. We'll have a release with it later this week.

Duplicate of #1304

**Comment by Saviq** [comment]
Duplicate of #1304
